skimage/filters/thresholding.py

```diff
@@ -727,14 +727,10 @@ def threshold_triangle(image, nbins=256):
     width /= norm
 
     # Maximize the length.
-    d = peak_height * arg_low_level - width * hist[arg_low_level]
```

Comment:
I need to have a look too. I actually reused the implementation in imagej. I also read the paper, and the description in there is pretty short.